source/css/sass/mixins/_decorations.scss

```diff
@@ -0,0 +1,14 @@
+@import "../mixins/logical-properties";
+@import "../variables/color";
+
+@mixin divider($dimension) {
+
+  @if index(("inline", "inline-start", "inline-end", "block", "block-start", "block-end"), $dimension) {
+    @include logical-property(border, $dimension, (1px solid $color-text-dividers,), $to-rem: false);
```

Comment:
`(1px solid $color-text-dividers,)` is a bit of pain, stops it being treated as a list in `logical-property` (and breaking on `inline` etc).